packages/pebble-web/src/components/Toggle.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { cx } from "emotion";
+import {
+  labelStyle,
+  selectedLabel,
+  inputStyle,
+  selectedSwitch,
+  switchStyle
+} from "./styles/Toggle.styles";
+
+interface ToggleProps {
+  initialValue?: boolean;
+  onChange?: (value: boolean) => void;
+}
+
+interface ToggleState {
+  value?: boolean;
```

Comment:
Why is this not a controlled component?
All of pebble's other components are controlled.